apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/service/NamespaceService.java

```diff
@@ -150,7 +155,7 @@ public NamespaceDTO loadNamespaceBaseInfo(String appId, Env env, String clusterN
         throw e;
       }
     }
-
+    NamespaceBOUtils.enrichPreferredUserName(namespaceBOs, this.userService::findPreferredUsernameMapByUserIds);
```

Comment:
how about we enrich the preferred user name info inside `transformNamespace2BO` right after `itemService.findItems(appId, env, clusterName, namespaceName)` so that outside callers don't need to care about this logic? 
by doing this, I think we could just resue the logic of `PreferredUsernameUtil#enrichPreferredUserName` and don't need to add the similar logic to `NamespaceBOUtils`